Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/SlideHeightTest.java

```diff
@@ -0,0 +1,48 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseTeleOp;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+@TeleOp(name = "SlideHeightTest")
+public class SlideHeightTest extends BaseTeleOp {
+
+    int[] heights = new int[] {0, 117, 238, 383, 537, Constants.SLIDE_LOW, Constants.SLIDE_MEDIUM, Constants.SLIDE_HIGH, Constants.SLIDE_TOP};
```

Comment:
Why is the first half magic numbers?

Reply:
@SJJCoding @valsei Could you take a look at this please?

Reply:
The heights are the newly measured stack positions, which should be a constant already.